pkg/utils/k8sutil.go

```diff
@@ -53,7 +53,8 @@ import (
 
 const (
 	controllerImage = "bitnami/kubeless-controller"
-	kafkaImage      = "wurstmeister/kafka"
+	kafkaImage      = "bitnami/kafka"
```

Comment:
@sebgoa I added a commit below: 

- use `KAFKA_CONTROLLER`, `ZOOKEEPER_CONTROLLER` and `KUBELESS_CONTROLLER` for custom images
- embed sha256 digest for kafka/zk images